stacks/port_udp.go

```diff
@@ -25,32 +25,36 @@ func (port udpPort) Port() uint16 { return port.port }
 // IsPendingHandling returns true if there are packet(s) pending handling.
 func (port *udpPort) IsPendingHandling() bool {
 	// return port.port != 0 && port.ihandler.isPendingHandling()
-	return port.port != 0 && port.ihandler.isPendingHandling()
+	return port.port != 0 && port.handler.isPendingHandling()
 }
 
 // HandleEth writes the socket's response into dst to be sent over an ethernet interface.
 // HandleEth can return 0 bytes written and a nil error to indicate no action must be taken.
 func (port *udpPort) HandleEth(dst []byte) (int, error) {
-	if port.ihandler == nil {
+	
+	//println("udpPort.HandleEth: ", string(dst)) //@@@REMOVE
+	if port.handler == nil {
 		panic("nil udp handler on port " + strconv.Itoa(int(port.port)))
 	}
-	return port.ihandler.send(dst)
+	
+	return port.handler.send(dst)
 }
 
-// Open sets the UDP handler and opens the port.
+// Open sets the UDP handler and opens the port. 
```

Comment:
Why are unchanged lines included in the diff. Are you formatting your Go code with gofmt? This should not appear in a Go diff since gofmt is pretty good at discarding whitespace noise. It worries me that there is some gomt-removable whitespace included on all your lines changed which would make the next commit *very* noisy, which we don't want.

Reply:
I have disabled an additional 3rd party linter that may have been responsible - i am using vscode and have the go extension installed